target/linux/ramips/patches-6.1/720-Revert-net-phy-simplify-phy_link_change-arguments.patch

```diff
@@ -107,9 +107,9 @@ still required by target/linux/ramips/files/drivers/net/ethernet/ralink/mdio.c
  	bool tx_pause, rx_pause;
 --- a/include/linux/phy.h
 +++ b/include/linux/phy.h
-@@ -703,7 +703,7 @@ struct phy_device {
- 	u8 mdix;
- 	u8 mdix_ctrl;
+@@ -723,7 +723,7 @@ struct phy_device {
+ 
+ 	int pma_extable;
  
 -	void (*phy_link_change)(struct phy_device *phydev, bool up);
 +	void (*phy_link_change)(struct phy_device *, bool up, bool do_carrier);
```

Comment:
This change looks suspicious, missing `phydev` param name?
I skipped this and the next (720 + 721) on mt7621, but guessing it is needed for the other subtargets. The patch was added with 5.10 https://github.com/openwrt/openwrt/commit/10267e17299806f9885d086147878f6c492cb904#diff-c9b12f819c9a81105654a075a5b9fe79652510bf665852290f8902bb9ec95367, and it looks like the carrier setting in mdio.c has always been there:
https://github.com/openwrt/openwrt/blob/ef98dc3b3e07f529831c3ebe719d1a98bd43b568/target/linux/ramips/files/drivers/net/ethernet/ralink/mdio.c#L93

Reply:
You mean I should ignore your comment here? And I should leave the patches like they are?